pkgs/development/python-modules/import-expression/default.nix

```
@@ -0,0 +1,42 @@
+{ lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  fetchpatch,
+  pytestCheckHook,
+  astunparse,
+  setuptools
+}:
+buildPythonPackage rec {
+  pname = "import-expression";
+  version = "1.1.4";
+  pyproject = true;
+  src = fetchFromGitHub {
+    owner = "ioistired";
+    repo = "import-expression-parser";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-mll2NePB7fthzltLOk6D9BgaDpH6GaW4psqcGun/0qM=";
+  };
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/ioistired/import-expression-parser/commit/3daf968c3163b64685aa529740e132f0df5ab262.patch";
+      hash = "sha256-2Ubv3onor2D26udZbDDMb3iNLopEIRnIcO/X6WUVmJU=";
+    })
+  ];
+  pythonImportsCheck = [
+    "import_expression"
+    "import_expression._codec"
+  ];
+  propagatedBuildInputs = [ astunparse ];
+  nativeBuildInputs = [ setuptools ];
+  nativeCheckInputs = [ pytestCheckHook ];
+  pytestFlagsArray = [ "tests.py" ];
```

```suggestion
  pyproject = true;

  src = fetchFromGitHub {
    owner = "ioistired";
    repo = "import-expression-parser";
    rev = "refs/tags/v${version}";
    hash = "sha256-mll2NePB7fthzltLOk6D9BgaDpH6GaW4psqcGun/0qM=";
  };

  patches = [
    (fetchpatch {
      url = "https://github.com/ioistired/import-expression-parser/commit/3daf968c3163b64685aa529740e132f0df5ab262.patch";
      hash = "sha256-2Ubv3onor2D26udZbDDMb3iNLopEIRnIcO/X6WUVmJU=";
    })
  ];

  nativeBuildInputs = [
    setuptools
  ];

  propagatedBuildInputs = [
    astunparse
  ];

  nativeCheckInputs = [
    pytestCheckHook
  ];

  pytestFlagsArray = [ "tests.py" ];

  pythonImportsCheck = [
    "import_expression"
    "import_expression._codec"
  ];

```
it would be great, if we could keep the more standard formatting used in python packages

---

I'll move the variables in the order presented here but it looks rather ugly having 3 lines for one package versus one line.

---

That's the standard formatting we use for python packages as it creates the least amount of diff lines on updates but whatever